### DISCUSSION TITLE
how to track actual image tokens processed - azure openai gpt4o - langfuse cloud free tier

### DISCUSSION THREAD

**Discussion by Hbmr9000** [langfuse/langfuse#4567] [Support] [answered]
hey!

i noticed that langfuse is propably parameterized wrongly as it only counts ~80 prompt tokens despite the image input resolution. the figures of the "actual" tokens i was reading of the azure dashboard, in which the gpt 4o is hosted via azure openai. does anybody else notice the difference?

thanks a lot!

**Comment by marcklingen** [comment, marked as answer]
thanks for sharing! @hassiebp is currently working on an improvement to token tracking for multi-modal models across langfuse

**Comment by hassiebp** [comment]
WIP PR to track: https://github.com/langfuse/langfuse/pull/4536
